src/views/Home.vue

```diff
@@ -1,36 +1,55 @@
 <template>
   <header
     id="home"
-    :style="{ 'background-image': headerImage }"
+    class="cover-image cover nested-img vh-100 bg-center center"
   >
-    <div class="logo">
+    <NavBar />
+    <div class="pt8 ma3 center measure-wide">
       <img
-        :src="headerLogo"
+        src="../assets/vector/bytie-gradient.svg"
```

Comment:
attempt using  `src="@/assets/vector/bytie-gradient.svg"`

Reply:
i did and it couldn't pathfind to it